lib/lte_link_control/Kconfig

```diff
@@ -5,7 +5,7 @@
 #
 
 menuconfig LTE_LINK_CONTROL
-	bool "nRF91 LTE Link control library"
+	bool "nRF91 LTE link control library"
```

Comment:
I'm thinking if we should or shouldn't have nRF91 mentioned here. I don't know if other libraries mention it...